README.md

```diff
@@ -1,2 +1,48 @@
-# wherobots-jdbc-driver
-JDBC Driver Implementation for the Wherobots Spatial SQL API
+# Wherobots Spatial SQL JDBC Driver
+
+This library provides a JDBC driver implementation for the Wherobots
+Spatial SQL API. It allows you to build Java/Scala/Kotlin applications
+that can interact with Wherobots and leverage the Spatial SQL and
+geospatial analytics capabilities of WherobotsDB.
+
+This JDBC driver is also directly usable from database applications like
+JetBrains's DataGrip or DBeaver.
+
+## Usage
+
+### As a library
+
+```gradle
+dependencies {
+  implementation 'com.wherobots.jdbc:wherobots-jdbc-driver:0.1.0'
+}
+```
+
+### In DataGrip
+
+TODO: add instructions for configuring a database connection with the
```

Comment:
Is this hard to set up?

Reply:
Not too much, but there are a few settings to get right given the limitations of our JDBC Driver implementation.